src/app_charts/base/cloud/registry-crd.yaml

```diff
@@ -54,3 +54,32 @@ spec:
               type: string
             project:
               type: string
+        status:
+          type: object
+          properties:
+            cloud:
+              type: object
+            robot:
+              type: object
+              properties:
+                updateTime:
+                  type: string
+                state:
+                  type: string
+                  enum:
+                    - UNDEFINED
+                    - UNAVAILABLE
+                    - AVAILABLE
+                    - EMERGENCY_STOP
+                    - ERROR
+                lastStateChangeTime:
+                  type: string
+                batteryPercentage:
+                  type: number
```

Comment:
I would prefer to not have batteryPercentage here. If one wants to do planning based on robot conditions, we'll probably need lots of predicates and this might be too much overload for the status api.

Reply:
That's fine with me. We probably should remove it from Go and protobuf data type too. What do you think?